commerce-batch/commerce-batch-service/service.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
+<!DOCTYPE service-builder PUBLIC "-//Liferay//DTD Service Builder 7.1.0//EN" "http://www.liferay.com/dtd/liferay-service-builder_7_1_0.dtd">
+
+<service-builder auto-namespace-tables="false" package-path="com.liferay.commerce.batch">
+	<author>Matija Petanjek</author>
+	<namespace>Commerce</namespace>
+	<entity local-service="true" name="CommerceBatchJobExecution" remote-service="true">
+
+		<!-- PK fields -->
+
+		<column name="commerceBatchJobExecutionId" primary="true" type="long" />
+
+		<!-- Audit fields -->
+
+		<column name="createDate" type="Date" />
+		<column name="modifiedDate" type="Date" />
+
+		<!-- Other fields -->
+
+		<column name="commerceBatchJobInstanceId" type="long" />
+		<column name="startTime" type="Date" />
+		<column name="endTime" type="Date" />
+		<column name="status" type="String" />
+		<column name="exitCode" type="String" />
+		<column name="exitMessage" type="String" />
+
+		<!-- Finder methods -->
+
+	</entity>
+	<entity local-service="true" name="CommerceBatchJobInstance" remote-service="true">
```

Comment:
CommerceBatchEntry

Reply:
@matijapetanjek as per our discussion, please update the PR to only have one table `CommerceBatchEntry` and merge the related fields in it: https://docs.google.com/document/d/1gCUYoc5wHTTOaSVArIePHbSrBdRsixmEux3l6dSvEuE/edit#heading=h.xq63w7dtts9b